ta/pkcs11/src/processing.c

```diff
@@ -88,6 +88,9 @@ void release_active_processing(struct pkcs11_session *session)
 		session->processing->tee_op_handle = TEE_HANDLE_NULL;
 	}
 
+	if (session->processing->extra_ctx)
```

Comment:
Testing `session->processing->extra_ctx` is not needed, `TEE_Free(NULL)` is valid as per GPD TEE API.

Reply:
Removed the check